src/closure-externs/node-externs.js

```diff
@@ -111,3 +111,13 @@ Buffer.prototype.slice = function(start, end) {};
  * @nosideeffects
  */
 Buffer.prototype.toString = function(encoding, start, end) {};
+
+/**
+ * @param {string} path
+ */
+var pathToFileURL = function(path) {};
+
+/**
+ * @param {URL|string} url
+ */
+var fileURLToPath = function(url) {};
```

Comment:
According to the docs seem to exist on the `url` module, rather than at the top level..?

Reply:
They are, but they're imported into the module's global scope (see `emcc.py`).

Reply:
But shouldn't closure be able to see that then?  It seems odd to me that we should need to declare these in this global location like this?  Perhaps a local annotation where they are declared/imported?

Reply:
It does seem odd. Maybe closure doesn't understand `await import('url')`?

Reply:
Indeed, it's a bit odd that this is needed. I think this was necessary when targeting both web/node as well as node only. That is:
```js
if (typeof process == 'object' &&
    typeof process.versions == 'object' &&
    typeof process.versions.node == 'string') {
  const { createRequire } = await import('module');
  var { pathToFileURL, fileURLToPath } = await import('url');
  var require = createRequire(import.meta.url);
}
```
(`-sENVIRONMENT=web,node`)
```js
import { createRequire } from 'module';
import { pathToFileURL, fileURLToPath } from 'url';
const require = createRequire(import.meta.url);
```
(`-sENVIRONMENT=node`)

Perhaps closure is run on an indeterminate file instead of the final JS file with these imports? Let me verify this.

Reply:
It seems that Closure indeed has some limitations with dynamic import expressions.
https://github.com/google/closure-compiler/wiki/JS-Modules#dynamic-import-expressions

I moved these externals to the correct file with commit 68377b6069d5bd3fad45965a66ee40bed9cecc46 and removed the need of these `import()`'s with commit bae5bf5bd5b7792ae93a085f96fc8a8637f52876.